nixos/modules/virtualisation/google-compute-config.nix

```
@@ -69,6 +69,30 @@ in
   # GC has 1460 MTU
   networking.interfaces.eth0.mtu = 1460;
 
+  systemd.services.fetch-nixops-ssh-keys = {
```

```suggestion
  systemd.services.fetch-instance-ssh-keys = {
```

This isn't nixops-specific per se. In theory, something else could also create these keys in the metadata. We could add a comment on top stating this is used by NixOps.

---

Yes, in theory, but I wanted the intention to be clear. If you're not running NixOps, seeing that a service called `fetch-instance-ssh-keys` has failed might make you a bit worried, whereas `fetch-nixops-ssh-keys` is likely less prone to.

---

The service wouldn't fail - I think the endpoint would just return an empty list. Really, this is NixOps independent metadata ssh keys, which could also have been clicked in the console, or configured by other tools. That it's configured by NixOps too is more of an implementation detail - we should move it to a comment.

---

I was going by https://cloud.google.com/compute/docs/storing-retrieving-metadata#default which lists the default attribute for SSH keys as `ssh-keys` instead of `sshKeys`, but maybe both are used to varying degrees :shrug: I'll rename the service and add a comment.